src/openforms/emails/tests/test_digest_functions.py

```diff
@@ -1361,6 +1361,74 @@ def test_invalid_logic_actions_with_exceptions_are_reported(self):
             invalid_logic_rules,
         )
 
+    @tag("gh-5271")
+    def test_invalid_logic_rules_with_reduce_and_accumulator_not_reported(self):
+        form = FormFactory.create()
+        FormVariableFactory.create(
+            form=form,
+            name="Pets",
+            key="pets",
+            user_defined=True,
+            data_type=FormVariableDataTypes.array,
+        )
+        FormLogicFactory(
+            form=form,
+            json_logic_trigger={
+                ">": [
+                    {
+                        "reduce": [
+                            {"var": "pets"},
+                            {"+": [{"var": "accumulator"}, 1]},
+                            0,
+                        ]
+                    },
+                    1,
+                ]
+            },
+        )
+
+        invalid_logic_rules = collect_invalid_logic_rules()
+
+        self.assertEqual(len(invalid_logic_rules), 0)
+
+    @tag("gh-5271")
+    def test_invalid_logic_rules_with_accumulator_without_reduce_not_reported(self):
```

Comment:
```suggestion
    def test_valid_logic_rules_with_accumulator_without_reduce_not_reported(self):
```
the logic rule is valid, so this test name is confusing otherwise